feedsOnChain.yaml

```diff
@@ -150,6 +150,34 @@ UMB-USD:
           fsym: UMB
           tsyms: USD
 
+# histo
+
+DAFI-USD-TWAP-1day:
+  discrepancy: 1.0
+  precision: 2
+  inputs:
+    - fetcher:
+        name: CoinmarketcapHistoHour
+        params:
+          symbol: DAFI
+          convert: USD
+          count: 25
```

Comment:
why 25, and not 24?

Reply:
Because Coinmarketcap counts the current hour as the first one and you can only ask for completed bars so you have to ask for n+1